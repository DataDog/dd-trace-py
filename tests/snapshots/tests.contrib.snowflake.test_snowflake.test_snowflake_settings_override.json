[[
  {
    "name": "sql.query",
    "service": "my-snowflake-svc",
    "resource": "select current_version();",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "sql",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "snowflake",
      "db.application": "PythonConnector",
      "db.name": "mock-db-name",
      "db.schema": "mock-schema-name",
      "db.system": "snowflake",
      "db.user": "mock-user",
      "db.warehouse": "mock-warehouse-name",
      "language": "python",
      "out.host": "mock-account.snowflakecomputing.com",
      "runtime-id": "f82097569d3141b8ba61409269818dec",
      "sfqid": "mocksfqid-29244a72-90a0-4309-9168-9dcf67721f6d",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "db.row_count": 1,
      "network.destination.port": 443,
      "process_id": 88705
    },
    "duration": 1283000,
    "start": 1667243933310187000
  }]]
