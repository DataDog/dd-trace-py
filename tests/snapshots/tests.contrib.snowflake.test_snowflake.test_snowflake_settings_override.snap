[[{"name" "snowflake.query"
   "service" "db"
   "resource" "select current_version();"
   "type" "sql"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1633558321134813900
   "duration" 3936500
   "meta" {"runtime-id" "d300825b8c4a43d8b95ee8f2add56bb5"
           "out.host" "mock-account.snowflakecomputing.com"
           "db.name" "mock-db-name"
           "db.user" "mock-user"
           "db.application" "PythonConnector"
           "db.schema" "mock-schema-name"
           "db.warehouse" "mock-warehouse-name"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 6454
              "out.port" 443
              "db.rowcount" 1
              "sql.rows" 1
              "_dd.tracer_kr" 1.0}}]]
